### ISSUE TITLE
focal_loss

### ISSUE THREAD

**Issue by dududukekeke** [yangdongchao/Tim-TSENet#2]
Hi, I saw the calculation of focal_Loss in your loss. py. Have you used this Loss function?Is it mainly for sound event detection? Is there any rule in parameter design?

**Comment by yangdongchao** [comment]
No, I try to use the focal loss in experiments. But I didnot get improvement, so that I donot use it anymore. It only aims for SED.
